client/ayon_cinema4d/startup/ayon_cinema4d.pyp

```diff
@@ -60,6 +64,8 @@ AYON_RESET_RESOLUTION_ID = 1064318
 AYON_RESET_COLORSPACE_ID = 1064320
 AYON_EXPERIMENTAL_TOOLS_ID = 1064319
 
+AYON_CONTEXT_LABEL = 1064309
```

Comment:
What did you base this number on @MustafaJafar - is this officially registered with Maxon?

Reply:
Honestly, I don't know. 

I think these are unique numbers to register the plugins/commands. I picked the smallest number - 1 in the pyp file.